src/Laravel/.gitattributes

```diff
@@ -1,4 +1,8 @@
 /.gitattributes export-ignore
 /.gitignore export-ignore
-/workbench export-ignore
+/phpstan.neon.dist export-ignore
+/phpunit.xml.dist export-ignore
+/README.md
```

Comment:
```suggestion
```

Let's keep the README? we don't remove it elsewhere.

Reply:
Currently, it only contains things that should be in `CONTRIBUTING.md`. Maybe should I rename it?